src/components/sbb-time-input/sbb-time-input.tsx

```diff
@@ -0,0 +1,157 @@
+import {
+  Component,
+  ComponentInterface,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  JSX,
+  Prop,
+} from '@stencil/core';
+import { forwardEventToHost } from '../../global/helpers/forward-event';
+import {
+  AccessibilityProperties,
+  getAccessibilityAttributeList,
+} from '../../global/interfaces/accessibility-properties';
+
+const REGEX_PATTERN = /[0-9]{3,4}/;
+const REGEX_GROUPS_WITH_COLON = /([0-9]{1,2})[.:,\-;_hH]?([0-9]{1,2})?/;
+const REGEX_GROUPS_WO_COLON = /([0-9]{1,2})([0-9]{2})/;
+
+@Component({
+  shadow: true,
+  styleUrl: 'sbb-time-input.scss',
+  tag: 'sbb-time-input',
+})
+export class SbbTimeInput implements ComponentInterface, AccessibilityProperties {
+  /** Value for the inner HTMLInputElement. */
+  @Prop({ mutable: true }) public value?: string = '';
+
+  /** Date value with the given time for the inner HTMLInputElement. */
+  @Prop({ mutable: true }) public valueAsDate?: Date;
```

Comment:
Have a `@Watch` that checks if the value is valid and reset to null, if it is not